src/wp-admin/site-editor.php

```diff
@@ -91,7 +91,7 @@ static function( $classes ) {
 	// Used in the HTML title tag.
 	$title = __( 'Editor (beta)' );
 
-	$active_global_styles_id = WP_Theme_JSON_Resolver::get_user_custom_post_type_id();
```

Comment:
I'm not very fond of the new name proposed as it loses meaning. If we want to remove the "cpt" reference, what about `get_user_global_styles_post_id`? Or something along those lines. As long as it's clear that the data stored is from the user I'm fine.

Reply:
Done in commit https://github.com/WordPress/wordpress-develop/pull/1957/commits/2180b6eb005ea73f03dfd24c1fd336ab23e5b687